### PR TITLE
New feature

### PR DESCRIPTION
**Pull request by mixicek**
Test description

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -116,10 +116,9 @@ dist
 # DynamoDB Local files
 .dynamodb/
```
Comment (CoffeeEyes28) [inline .gitignore:118]: Are we sure?

**Review by CoffeeEyes28 — CHANGES_REQUESTED** [review]
Lets look at line 118

File: .gitignore (hunk 1 above)
Comment (mixicek) [reply .gitignore:118]: @CoffeeEyes28, thanks for the comment. I made the change. Could you please review again?

Comment (CoffeeEyes28) [reply .gitignore:118]: lgtm

**Review by CoffeeEyes28 — APPROVED** [review]
lgtm

**Pull request merged**